### PR TITLE
Feature: wgbs workflow

### PR DESCRIPTION
**Pull request by JenkoB**
@jkokosar review

### CODE REVIEW

File: resolwe_bio/processes/workflows/wgbs.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - name: reads
+      label: Input reads
+      type: data:reads:fastq
+    - name: genome
+      label: genome
+      type: data:genome:fasta
+    - name: genome_identifier
+      label: The UCSC Genome Browser identifier
+      type: basic:string
+      allow_custom_choice: true
```
Comment (jkokosar) [inline resolwe_bio/processes/workflows/wgbs.yml]: What happens if one enters a string that is not on a list?

File: resolwe_bio/processes/workflows/wgbs.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        - label: mm8
+          value: mm8
+        - label: mm9
+          value: mm9
+        - label: mm10
+          value: mm10
+    - name: threads
+      label: Threads
+      type: basic:integer
+      default: 3
```
Comment (jkokosar) [inline resolwe_bio/processes/workflows/wgbs.yml]: Change this to 1 and than pass a higher number as a workflow parameter where required

File: resolwe_bio/processes/workflows/wgbs.yml (hunk 1 above)
Comment (JenkoB) [inline resolwe_bio/processes/workflows/wgbs.yml]: I am not clearly sure how this parameter is used in the tool code, but the code is working normally if you put other genomes_indentifier for example rn6.

**Comment by mstajdohar** [comment]
@JenkoB Rebase

**Comment by JenkoB** [comment]
Rebased, can be merged

**Pull request merged**